src/plugin/kubearmor.go

```diff
@@ -469,17 +469,21 @@ func StartKubeArmorRelay(StopChan chan struct{}, cfg types.ConfigKubeArmorRelay)
 				}
 
 				kubearmorLog := pb.Log{
-					ClusterName:   res.ClusterName,
-					ContainerName: res.ContainerName,
-					HostName:      res.HostName,
-					NamespaceName: res.NamespaceName,
-					PodName:       res.PodName,
-					Source:        res.Source,
-					Operation:     res.Operation,
-					Resource:      res.Resource,
-					Data:          res.Data,
-					Result:        res.Result,
-					Type:          res.Type,
+					ClusterName:       res.ClusterName,
+					ContainerName:     res.ContainerName,
+					ContainerID:       res.ContainerID,
```

Comment:
Cant we directly assign `res` to `kubearmorLog`? Like what we do in the case of telemetry events above.

Reply:
We need policy-name to be part of this.